packages/gatsby-theme-apollo-core/src/utils/colors.js

```diff
@@ -3,8 +3,8 @@ const {colors} = require('@apollo/space-kit/colors');
 exports.colors = {
   primary: colors.indigo.dark,
   primaryLight: colors.indigo.lighter,
-  secondary: colors.pink.base,
-  tertiary: colors.teal.dark,
+  secondary: colors.pink.darker,
```

Comment:
Looks awesome! Thanks @StephenBarlow. Since we only step from `dark` to `darker` below, should we step up one time here too, from `base` to `dark` instead of `darker`?